pkg/reader/price_reader.go

```diff
@@ -184,7 +190,7 @@ func (pr *priceReader) getFeedDecimals(
 ) (uint8, error) {
 	var decimals uint8
 	if err :=
-		pr.feedChainReader.GetLatestValue(
+		pr.feedChainReader().GetLatestValue(
```

Comment:
Can be nil, is this covered by the nil check above?

Reply:
Yes, it's covered by the nil check above, doing extra won't hurt though.